supertokens_python/recipe/jwt/recipe_implementation.py

```diff
@@ -52,7 +52,7 @@ async def create_jwt(
         data = {
             "payload": payload,
             "validity": validity_seconds,
-            "use_static_signing_key": use_static_signing_key is not False,
+            "useStaticSigningKey": use_static_signing_key is not False,
```

Comment:
Is the CHANGELOG going to be updated in this PR?

Reply:
Yes. TODO

Reply:
Done